app/src/main/java/com/example/nockanakalinowej/LevelMenager.java

```diff
@@ -50,11 +50,11 @@ public LvlMng(Level context, int LvlNo){
         setLayout(setKafels(LvlNo));
     }
 
-    protected void setLayout(int kafelNo){
-        final ImageButton[] buttArr = new ImageButton[kafelNo];
-        int edge = kafelEdge(kflamountx, kflamounty);
-        lvlLayout = new ConstraintLayout(window);
-        lvlLayout.setId(R.id.Level_layout);
+    protected void setLayout(int tileAmount){
+        final ImageButton[] buttArr = new ImageButton[tileAmount];
```

Comment:
Suggest to change to buttonsArray

Reply:
and done

Reply:
well buttonArr is kinda different than buttonsArray, but whatever